detection_rules/etc/version.lock.json

```diff
@@ -7617,15 +7631,15 @@
       "7.16": {
         "max_allowable_version": 99,
         "rule_name": "System Log File Deletion",
-        "sha256": "32f9c84c5cce449a17ac218b0c36c04bf7a4b6e603bdc57b4ab1163a8a3082d5",
+        "sha256": "9a47a157326055c14e4487b55a906009c79e7e0b45fb280ccbef121b35e74e8e",
```

Comment:
Forked rule changed when all rules are min-stacked at 8.3 and thus changes would not backport to this branch.